src/www/services_unbound_host_edit.php

```diff
@@ -58,9 +59,17 @@ function hosts_sort() {
 	$id = $_POST['id'];
 
 if (isset($id) && $a_hosts[$id]) {
+/* Backwards compatibility for records created before introducing different RR types. */
+	if (!isset($a_hosts[$id]['rr'])) {
+		$a_hosts[$id]['rr'] = 'A';
+	}
+
 	$pconfig['host'] = $a_hosts[$id]['host'];
 	$pconfig['domain'] = $a_hosts[$id]['domain'];
+	$pconfig['rr'] = $a_hosts[$id]['rr'];
 	$pconfig['ip'] = $a_hosts[$id]['ip'];
+	$pconfig['mxprio'] = $a_hosts[$id]['mxprio'];
+	$pconfig['mx'] = $a_hosts[$id]['mx'];
```

Comment:
@fichtner I'm also not happy with that restriction, but did not change it intentionally yet, since it would change the current behavior of the host overrides. Please see my last comment on #511 regarding that.